x/ibctransfer/module.go

```diff
@@ -124,13 +132,14 @@ func (am AppModule) OnTimeoutPacket(ctx sdk.Context, packet types.Packet) (*sdk.
 	return am.cosmosAppModule.OnTimeoutPacket(ctx, packet)
```

Comment:
I think this is meant to be cosmosAppModule.OnAcknowledgementPacket()

Reply:
Isn't it correct?